addons/nuxeo-binary-metadata/src/main/java/org/nuxeo/binary/metadata/internals/MetadataMappingDescriptor.java

```diff
@@ -86,7 +91,19 @@ public String getBlobXPath() {
         return blobXPath;
     }
 
+    /**
+     * @deprecated since 11.1, use {@link #ignorePrefix()}
+     */
+    @Deprecated(since = "11.1")
     public Boolean getIgnorePrefix() {
-        return ignorePrefix;
+        return !Boolean.FALSE.equals(ignorePrefix);
```

Comment:
`getIgnorePrefix`, the deprecated one, should just `return ignorePrefix()` by delegating to the new one.

Reply:
👍